drivers/gpio/gpio_stm32.c

```diff
@@ -561,6 +576,65 @@ static const struct gpio_driver_api gpio_stm32_driver = {
 	.manage_callback = gpio_stm32_manage_callback,
 };
 
+#ifdef CONFIG_PM_DEVICE
+static uint32_t gpio_stm32_get_power_state(const struct device *dev)
+{
+	struct gpio_stm32_data *data = dev->data;
+
+	return data->power_state;
+}
+
+static int gpio_stm32_set_power_state(const struct device *dev,
+					      uint32_t new_state)
+{
+	struct gpio_stm32_data *data = dev->data;
+	int ret = 0;
+
+	if (new_state == PM_DEVICE_ACTIVE_STATE) {
```

Comment:
where  `PM_DEVICE_ACTIVE_STATE`  differs from `DEVICE_PM_ACTIVE_STATE` (as defined by the include/device.h) ?

Reply:
@FRASTM all `DEVICE_PM` macros have been moved to the `PM_DEVICE` namespace

Reply:
PM_DEVICE_ACTIVE_STATE is the new DEVICE_PM_ACTIVE_STATE in latest master.
Change occured in #34698

Reply:
ok